govcd/api_vcd_test.go

```diff
@@ -566,8 +569,13 @@ func (vcd *TestVCD) SetUpSuite(check *C) {
 		panic(err)
 	}
 
-	// This is set explicitly to make sure that catalog is using storage profile from correct VDC
-	setCatalogStorageProfile(vcd, check)
+	// configure NSX-T VDC for convenience if it is specified in configuration
+	if config.VCD.Nsxt.Vdc != "" {
+		vcd.nsxtVdc, err = vcd.org.GetVDCByName(config.VCD.Nsxt.Vdc, false)
+		if err != nil {
+			panic(fmt.Errorf("error geting NSX-T Vdc '%s': %s", config.VCD.Nsxt.Vdc, err))
```

Comment:
Nit:

```suggestion
			panic(fmt.Errorf("error geting NSX-T VDC '%s': %s", config.VCD.Nsxt.Vdc, err))
```

Reply:
Changed. I will get there one day. I do remember it most of the times now.